alibi_detect/saving/loading.py

```diff
@@ -186,8 +182,10 @@ def _load_kernel_config(cfg: dict, backend: str = Framework.TENSORFLOW) -> Calla
     """
     if backend == Framework.TENSORFLOW:
         kernel = load_kernel_config_tf(cfg)
-    else:
+    elif backend == Framework.PYTORCH:
         kernel = load_kernel_config_pt(cfg)
+    else:  # backend=='keops'
+        kernel = load_kernel_config_ke(cfg)
```

Comment:
Not a massive deal but rather than:

```py
if backend == Framework.TENSORFLOW:
        kernel = load_kernel_config_tf(cfg)
    elif backend == Framework.PYTORCH:
        kernel = load_kernel_config_pt(cfg)
    else:  # backend=='keops'
        kernel = load_kernel_config_ke(cfg)
```

we might be able to do:

```py
return {
    Framework.TENSORFLOW: load_kernel_config_tf,
    Framework.PYTORCH: load_kernel_config_pt,
    Framework.KEOPS: load_kernel_config_ko
}[backend]()
```

Reply:
@mauicv happy to make this change, do agree it looks nice. My only concern is if any of the `load_kernel_config_` functions needed kwarg's for whatever reason later on. Could always change it back then though...

Reply:
Yeah fair enough

Reply:
@mauicv so I can merge this now, I have added this change as a follow-up in https://github.com/SeldonIO/alibi-detect/issues/683.